domain/src/main/java/com.pyshankov.microservices.domain/Order.java

```diff
@@ -1,18 +1,22 @@
 package com.pyshankov.microservices.domain;
 
+import org.springframework.data.annotation.Id;
+
 import java.math.BigDecimal;
 import java.time.LocalDateTime;
 import java.util.List;
 
 /**
  * Created by pyshankov on 4/29/18.
  */
+
 public class Order {
+    @Id
     private String id;
     private LocalDateTime createDate;
     private List<Product> products;
     private BigDecimal price;
-
+    private User user;
```

Comment:
I think it will be better to store only userName there instead of the hole User object(userName is email)